configs/opencv-ovms/gst_capi/pipelines/face_detection/main.cpp

```diff
@@ -281,7 +281,7 @@ class FaceDetection0005 : public ObjectDetectionInterface {
 public:
 
     FaceDetection0005() {
-        confidence_threshold = .5;
+        confidence_threshold = .9;
```

Comment:
it would be great if we can have the confidence threshold as an argument. Because depending on the video, we would like to tweak this value.

Reply:
Also, .9 is too high to have it as a default.